src/lib/yang/binary.lua

```diff
@@ -495,6 +536,7 @@ function selftest()
       assert(routing_table:lookup_ptr(key).value.port == 10)
       key.addr = util.ipv4_pton('3.4.5.6')
       assert(routing_table:lookup_ptr(key).value.port == 2)
+      assert(data.next_hop.ipv4 == util.ipv4_pton('5.6.7.8'))
```

Comment:
Would be useful to print out at which round this is failing: when loaded from source, written to binary and reloaded, or written from loaded and then reloaded?